packages/addon/public/token-bridge.js

```diff
@@ -0,0 +1,79 @@
+const ALLOWED_ORIGINS = new Set([
+  "https://auth-stage.tb.pro",
+  "http://localhost:5173",           // dev
+  "http://127.0.0.1:5173"            // dev
+]);
+
+window.postMessage({ type: "TB/BRIDGE_READY" }, window.location.origin);
+console.log(`[🌉 token-bridge] the token bridge has loaded.`);
+
+// Visual cue, make sure to remove.
+const tag = document.createElement("div");
+tag.textContent = "✅ Content script injected";
+Object.assign(tag.style, {
+  position: "fixed", zIndex: 999999, inset: "8px auto auto 8px",
+  padding: "6px 10px", background: "lime", color: "black",
+  fontFamily: "monospace", boxShadow: "0 2px 8px rgba(0,0,0,.25)"
+});
+document.documentElement.appendChild(tag);
```

Comment:
This draws an ugly, green rectangle on the Vue web app, just so we know that the content script is added.